Makefile

```diff
@@ -1,7 +1,7 @@
 test:
 	py.test
 testcov:
-	py.test --cov=ideascube/ --cov-report=term-missing
+	py.test --cov=ideascube/ --cov-report=term-missing --migrations
```

Comment:
Just to not leave this hanging: this is now done in the latest push to this branch.